examples/mean-reversion.go

```diff
@@ -0,0 +1,255 @@
+package main
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/alpacahq/alpaca-trade-api-go/alpaca"
+	"github.com/alpacahq/alpaca-trade-api-go/common"
+	"github.com/shopspring/decimal"
+)
+
+type alpacaClientContainer struct {
+	client         *alpaca.Client
+	closingPrices  []float64
+	runningAverage float64
+	lastOrder      string
+	timeToClose    float64
+	stock          string
+}
+
+var alpacaClient alpacaClientContainer
+
+func init() {
+	API_KEY := "API_KEY"
+	API_SECRET := "API_SECRET"
+	BASE_URL := "https://paper-api.alpaca.markets"
+
+	os.Setenv(common.EnvApiKeyID, API_KEY)
+	os.Setenv(common.EnvApiSecretKey, API_SECRET)
+	alpaca.SetBaseUrl(BASE_URL)
+
+	alpacaClient = alpacaClientContainer{
+		alpaca.NewClient(common.Credentials()),
+		[]float64{},
+		0.0,
+		"",
+		0.0,
+		"AAPL",
+	}
+}
+
+func main() {
+	// First, cancel any existing orders so they don't impact our buying power.
+	status, until, limit := "open", time.Now(), 100
+	orders, _ := alpacaClient.client.ListOrders(&status, &until, &limit)
+	for _, order := range orders {
+		_ = alpacaClient.client.CancelOrder(order.ID)
+	}
+
+	// Wait for market to open
+	fmt.Println("Waiting for market to open...")
+	for {
+		isOpen := alpacaClient.awaitMarketOpen()
+		if isOpen {
+			break
+		}
+		time.Sleep(2000 * time.Millisecond)
+	}
+	fmt.Println("Market Opened.")
+
+	// Get the running average of prices of the last 20 minutes, waiting until we have 20 bars from market open.
```

Comment:
The comment for this block should probably just be `Wait until 20 bars of data since market open have been collected`, and then the part about the running average can be moved down to the next block.